docs/api/expect.md

```diff
@@ -1405,3 +1405,70 @@ Don't forget to include the ambient declaration file in your `tsconfig.json`.
 :::tip
 If you want to know more, checkout [guide on extending matchers](/guide/extending-matchers).
 :::
+
+## expect.addEqualityTesters <Badge type="info">1.2.0+</Badge>
+
+- **Type:** `(tester: Array<Tester>) => void`
+
+You can use this method to define custom matchers to test if two object equals are equal. It is compatible with Jest's `expect.extend`.
+
+```ts
+class AnagramComparator {
```

Comment:
I think It's a little more kind to have copy-paste-able code snippets for users.
How about putting everything in a single block with required `import`?

```tsx
import { expect, test } from "vitest"

...

// I think you can drop `Tester` type since explicitly having a type here should help users enough
// btw, it should be also named "areAnagramsEqual" instead of "areObjectsEqual"?
const areAnagramsEqual = (
  a: unknown,
  b: unknown,
): boolean | undefined => {
   ...
}

expect.addEqualityTesters([areAnagramsEqual])  

test("...", () => {
  ...
})
```

Reply:
Done.

Reply:
Thanks for the update!

How about removing `: Tester` typing? I think it's only exposed from `@vitest/expect` package, but for this doc example, it's fine to remove the reference of `Tester` since having explicit type `(a: unknown, b: unknown): boolean | undefined` tells enough about the expected usage.

Reply:
Removed it, thanks.